cinema/views.py

```diff
@@ -167,6 +184,24 @@ def get_serializer_class(self):
 
         return MovieSessionSerializer
 
+    @extend_schema(
+        parameters=[
+            OpenApiParameter(
+                "date",
+                type={"type": "OpenApiParameter.DATE"},
```

Comment:
```suggestion
                type={"type": OpenApiParameter.DATE},
```

Reply:
I've tried this from the beginning, but I couldn't run app without quotes

Reply:
try 

`from drf_spectacular.utils import extend_schema, OpenApiParameter
`

Reply:
are you getting any error?

Reply:
yes,   File "D:\python_work\Django REST\py-documentation-and-tests\cinema\views.py", line 191, in MovieSessionViewSet
    type={"type": OpenApiParameter.DATE},
                  ^^^^^^^^^^^^^^^^^^^^^
AttributeError: type object 'OpenApiParameter' has no attribute 'DATE'